requirements/check-avail-strategies.py

```diff
@@ -0,0 +1,9 @@
+import bagua
+import deepspeed
+import fairscale
+import horovod.torch
```

Comment:
```suggestion
import horovod.torch; horovod.torch.nccl_built()
```

Also, does this need an `assert`? Doesn't `nccl_built` return a bool?